src/plugin.js

```diff
@@ -40,11 +40,11 @@ class LivrPlugin {
     Vue.mixin(mixin);
   }
 
-  initVM() {
+  initVM({ errorBagName, fieldsBagName }) {
```

Comment:
Is that a case for default params?

Reply:
There is already default parameters for this values on `config.js`